src/main/java/org/apache/brooklyn/cloudfoundry/location/CloudFoundryPaasLocation.java

```diff
@@ -585,24 +587,13 @@ private void init(Map<String, Object> params) {
         }
 
         private Optional<VcapDescription> findVcapDescription(String instanceName) {
```

Comment:
call it `tryFindVcapDescription` and use `Iterables.tryFind(vcapServices.values(), new Predicate() { return instanceName.equals(value.getInstanceName()); } );